src/lib/util/constants/actions.ts

```diff
@@ -1,23 +1,39 @@
-type ActionDefinition = {
+import {
+    ActionDefinitionParameter,
+    toggleFeatureParameters,
+} from './action-parameters';
+
+export type ActionDefinition = {
     label: string;
+    description?: string;
+    category?: string;
     permissions: string[];
+    parameters: ActionDefinitionParameter[];
+    // TODO: Remove this in favor of parameters (filter by !optional)
```

Comment:
We're only keeping this so Enterprise does not break. We'll remove this in a future PR.